docs/user/metrics/timing_distribution.md

```diff
@@ -11,6 +11,8 @@ That is, the function from a value \\( x \\) to a bucket index is:
 
 This makes them suitable for measuring timings on a number of time scales without any configuration.
 
+> **Note** Check out how this bucketing algorithm would behave on our [Histogram Simulator](../../appendix/histograms.html)
```

Comment:
We can add params to this URL to open the simulator on a plot that matches the bucketing algorithm for the timing distribution, but I honestly was not sure what these params were. We would need, lowerBound, upperBound and bucketCount.